### DISCUSSION TITLE
Does the standalone builds support non jack audio output ?

### DISCUSSION THREAD

**Discussion by SeleDreams** [DISTRHO/DPF#481] [Q&A]
Hi, there was something i've been wondering about standalone, basically I'd like to make a software that works both as a standalone and a vst, that's why standalone is also a focus for me, however, i'd want it to be a very simple install for end users not using jack and all, so it should also work on simple pulseaudio on linux and wasapi/mme etc on windows, even if there's more latency in my use case it wouldn't be a big deal
is this supported by DPF ? if not i feel like it would be a pretty good addition

**Comment by falkTX** [comment]
It is yes. There is RtAudio in place and used by default for the cases where JACK is not available.
With the caveat that it will always connect to the default audio interface, with no chance to configure this besides buffer size.

See the calls in https://github.com/DISTRHO/DPF/blob/main/distrho/DistrhoStandaloneUtils.hpp
Still a bit experimental, so try it out and let me know how that works.

**Comment by SeleDreams** [comment]
I see, thanks, I'll look into it
would be a good addition to find a way to modify at runtime the interface, but it's already a good thing that it's there at all
